test/Claim/IsValidClaim.ts

```diff
@@ -0,0 +1,39 @@
+/* tslint:disable:no-relative-imports */
```

Comment:
Is this a unit test? If so, let's move it to `src/Claim.test.ts`.

Reply:
We already had in place unit tests for poet-js with this structure. We're sticking with it in this PR. We can move all of them in a separate PR

Reply:
I agree with @lautarodragan, we have to do it in a separate PR

Reply:
Create a file for it? There are some other tests stored in the same folder as I made this one.